ros2cli/ros2cli/verb/daemon/start.py

```diff
@@ -26,6 +26,10 @@ def add_arguments(self, parser, cli_name):
         parser.add_argument(
             '--debug', '-d', action='store_true',
             help='Print debug messages')
+        parser.add_argument(
+            '--timeout', '-t', default=2 * 60 * 60,
+            help='Terminate daemon after integer t seconds of inactivity. \
+            Default of 7200 seconds(2 hours)')
```

Comment:
@cnboonhan94 IIRC there's no need to introduce the default value in the help string, the parser usage string will include it.